apps/pano/src/features/comment/MoreOptionsDropdown.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+  FacebookShare,
+  GappedBox,
+  IconButton,
+  RedditShare,
+  styled,
+  Toast,
+  TwitterShare,
+} from "@kampus/ui";
+import { DotsHorizontalIcon } from "@radix-ui/react-icons";
+import { useNavigate } from "@remix-run/react";
+import type { FC } from "react";
+import { useEffect, useState } from "react";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import type { Comment } from "~/models/comment.server";
+import { getExternalCommentURL } from "~/utils";
+
+const DotsButton = styled(IconButton, {
+  borderRadius: 5,
+  height: 25,
+  width: 25,
+  color: "$gray11",
+});
+
+const Item = styled(DropdownMenuItem, {
+  minWidth: 0,
+  // padding: "$1",
+});
+
+interface Props {
+  comment: Comment;
+}
+
+export const MoreOptionsDropdown: FC<Props> = ({ comment }) => {
+  const [openToast, setOpenToast] = useState(false);
+  const [externalCommentUrl, setExternalCommentUrl] = useState("");
+
+  useEffect(() => {
+    setExternalCommentUrl(getExternalCommentURL(comment));
+  }, [comment]);
+
+  return (
+    <>
+      <DropdownMenu>
+        <DropdownMenuTrigger asChild>
+          <DotsButton color="transparent" aria-label="Daha fazla seçenek">
+            <DotsHorizontalIcon />
+          </DotsButton>
+        </DropdownMenuTrigger>
+        <DropdownMenuContent>
+          <MoreOptionsShareButtons
+            commentUrl={externalCommentUrl}
+            openToast={setOpenToast}
+          />
+        </DropdownMenuContent>
+      </DropdownMenu>
+      <Toast
+        open={openToast}
+        setOpen={setOpenToast}
+        duration={5000}
+        title="Link kopyalandı"
+      />
+    </>
+  );
+};
+
+const SocialMediaShareButtons = ({ commentUrl }: { commentUrl: string }) => {
```

Comment:
It feels like we need to make this a generic component, but we can do that later in a follow up PR.